recipe/meta.yaml

```diff
@@ -1,25 +1,24 @@
 {% set version = "1.7.3" %}
-{% set hash = "33b878c28cf2430818e7f8c2f5705c6954fb331a109d051bad0f2e5855cbb4a4" %}
```

Comment:
Why this change?

Reply:
That is the new standard in `conda-skeleton` and the recipes updates when I regenerate it.

Reply:
I always found it more useful this way so that the two things that I have to update when I release are at the top.

Reply:
Depends on the turnaround time for the bot. Usually for doctr I want to get releases out to conda right away.